html/partials/503.shtml

```diff
@@ -1,9 +1,10 @@
+<h2>503 Service unavailable</h2>
 <p>
-  Sorry, we are not currently able to fulfil your response. There are
-  two main situation in which this can occur:
+  Sorry, we can’t carry out your request at the moment. There are
+  two main situations in which this can occur:
```

Comment:
Sorry, we can't carry your request.

We think:

"Sorry, we can’t carry out your request at the moment. 
 
Please try again or come back later."